akka-actor/src/main/scala/akka/actor/ActorCell.scala

```diff
@@ -521,8 +521,8 @@ private[akka] class ActorCell(
           case NoMessage                                                    => // only here to suppress warning
         }
       } catch handleNonFatalOrInterruptedException { e =>
-        handleInvokeFailure(Nil, e)
-      }
+          handleInvokeFailure(Nil, e)
+        }
```

Comment:
Curious why the last bracket is indented

Reply:
This is a result of the `binPack` features, specifically `binPack.indentCallSiteOnce`. This was a feature that was added just now in scalafmt to fix other diffs, see discussion at https://github.com/mdedetrich/akka-apache/pull/2#discussion_r1002684029